alibi_detect/od/pytorch/knn.py

```diff
@@ -92,7 +91,7 @@ def score(self, x: torch.Tensor) -> torch.Tensor:
         all_knn_dists = bot_k_dists.values[:, self.ks-1]
         return all_knn_dists if self.ensemble else all_knn_dists[:, 0]
 
-    def fit(self, x_ref: torch.Tensor) -> Self:
+    def fit(self, x_ref: torch.Tensor):
```

Comment:
`Self ` has gone missing?

Reply:
I removed `return self` from the `KNNTorch` fit method so it's no longer needed!